yarn.lock

```diff
@@ -8455,18 +8465,6 @@ nanoid@^3.3.4:
   resolved "https://registry.yarnpkg.com/nanoid/-/nanoid-3.3.4.tgz#730b67e3cd09e2deacf03c027c81c9d9dbc5e8ab"
   integrity sha512-MqBkQh/OHTS2egovRtLk45wEyNXwF+cokD+1YPf9u5VfJiRdAiRwB2froX5Co9Rh20xs4siNPm8naNotSD6RBw==
 
-<<<<<<< HEAD
```

Comment:
Shouldn't this just be fixed separately (and ASAP)? Doesn't really have anything to do with the feature.